processor/geoipprocessor/internal/provider/config.go

```diff
@@ -0,0 +1,11 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Can we remove this file and the whole `internal/provider` directory in this initial PR?